bbchallenge-paper.tex

```diff
@@ -396,12 +396,12 @@ \subsection{Discussion}\label{sec:intro:discuss}
 
 
 \begin{conjecture}[Antihydra does not halt]
-    Consider the Collatz-like map $H: \mathbb{N} \to \mathbb{N}$ defined by $H(x) = 3\frac{x}{2}$ if $x$ is even and $H(x) = 3\frac{x-1}{2}$ if $x$ is odd. Iterating $H$ from $x=8$, there are never (strictly) more than twice as many odd numbers as even numbers.
+    Consider the Collatz-like map $H: \mathbb{N} \to \mathbb{N}$ defined by $H(x) = 3\frac{x}{2}$ if $x$ is even and $H(x) = 3\frac{x-1}{2}$ if $x$ is odd. Define $a_0 = 8,\ a_{n+1} = H(a_n)$, the sequence of iterates of $H$ starting from $x = 8$. No finite prefix of this sequence contains (strictly) more than twice as many odd numbers as even numbers.
 \end{conjecture}
 
 Using Conway's terminology, the conjecture \textit{probviously}\footnote{Portmanteau of the words probabilistic and obvious.} holds because a probabilistic \href{https://wiki.bbchallenge.org/w/index.php?title=Antihydra}{analysis} of the Turing machine suggests that its probability of ever halting is minuscule, namely, $\smash{\left(\frac{\sqrt{5}-1}{2}\right)}^{1073720885}\approx 4.841\times 10^{-224394395}$. However, properly proving that Antihydra does not halt is believed mathematically hard, because of resemblance with the notoriously hard Collatz conjecture \cite{LagariasCollatz}: Antihydra is a Cryptid. Also, the map $H$ was studied before Antihydra was discovered and is known to not generate Sturmian words \cite{DUBICKAS_2009}.
 
-Antihydra is the \textit{smallest}\footnote{We refrained from saying the \textit{simplest} since there are machines with open halting problems which are simpler to solve.} open problem in mathematics, on the Busy Beaver scale. Or, to be exact, one of the smallest since there are other 6-state Cryptids and here is one of them formulated as a \href{https://wiki.bbchallenge.org/wiki/Beaver_Math_Olympiad}{Beaver Math Olympiad} (BMO) problem:
+Antihydra is the \textit{smallest}\footnote{We refrained from saying the \textit{simplest} since there are machines with open halting problems which are simpler to solve.} open problem in mathematics, on the Busy Beaver scale. Or, to be exact, one of the smallest, since other 6-state Cryptids exist. Here, we present one of them formulated, perhaps somewhat humorously, as a \href{https://wiki.bbchallenge.org/wiki/Beaver_Math_Olympiad}{Beaver Math Olympiad} (BMO) problem:
```

Comment:
This formulation is not ideal because we don't understand if "somewhat humorously" refers to Antihydra or the BMO.